deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/TwinParser.java

```diff
@@ -107,7 +107,7 @@ public void setReportedCallback(TwinChangedCallback onReportedCallback)
      */
     public void setTagsCallback(TwinChangedCallback onTagsCallback)
     {
-        TwinParser.onTagsCallback = onTagsCallback;
```

Comment:
This was another warning caught by Spotbugs. This was unusual of us to have a static field with an instance method setter. This class is deleted in v2 though, and is unused currently, so this change won't impact anyone